### PR TITLE
Update CHANGELOG v1.0

### PR DESCRIPTION
**Pull request by nopcoder**

### CODE REVIEW

**Review by itaiad200 — APPROVED** [review]
Where do we use @ozkatz change log?

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,7 +4,48 @@
...
+This is a huge milestone for the lakeFS maintainers, contributors and community and we can't be more excited!
+
+## Benefits
+
+1.0 is more than just a cosmetic change - it includes a few important benefits and guarantees to anyone running or using lakeFS:
+
+* lakeFS 1.x ensures both [backwards]() and [forwards]() compatiblity with any future 1.x release, as per [Semver](https://semver.org/).
+* Not just the [lakeFS Open API](https://docs.lakefs.io/reference/api.html): This is also true for the [new lakeFS Python SDK](https://docs.lakefs.io/integrations/python.html#using-the-lakefs-sdk), the [Java SDK](https://central.sonatype.com/artifact/io.lakefs/sdk?smo=true) and the [lakectl CLI](https://docs.lakefs.io/reference/cli.html) tool.
+* Bug fixes will be released for lakeFS 1.x even after the release of future major version, and at least until October 2024
+* Security fixes will be released (and if needed, backported) ven after the release of future major version, and at least until June 2025(!).
```
Comment (N-o-Z) [inline CHANGELOG.md:20]: ven?

**Comment by nopcoder** [comment]
> Where do we use @ozkatz change log?

Updated with the latest information and link to the code migration guide.

**Review by N-o-Z — COMMENTED** [review]
Forgot to submit

File: CHANGELOG.md (hunk 1 above)
Comment (nopcoder) [reply CHANGELOG.md:20]: fixing

**Pull request merged**